osu.Game.Tests/Visual/Gameplay/TestSceneHitErrorMeter.cs

```diff
@@ -155,16 +155,16 @@ public void TestProcessingWhileHidden()
                     hitErrorMeter.Hide();
             });
 
-            AddRepeatStep("hit", () => newJudgement(), ColourHitErrorMeter.MAX_DISPLAYED_JUDGEMENTS * 2);
+            AddRepeatStep("hit", () => newJudgement(), 10);
```

Comment:
This looks wrong? You've made the thing this was testing no longer tested. It needs to be double the count below.

This is why we use constants. Because you should see the `*2` and understand the purpose it serves.

Please add a local `const` in this test and restore the previous logic.